index.html

```diff
@@ -1198,23 +1202,40 @@
         NDEFRecordData data;
       };
     </pre>
+    <p>
+      <a>NDEFRecord</a> objects have an associated <dfn id="ndefrecord-bytes">bytes</dfn>
+      (a <a>byte sequence</a>) set on creation that represents the payload data of the
```

Comment:
comma before set, otherwise it sounds like >bytes set< 

Also, bytes is plural, so 'an bytes' is wrong. Just drop an, I would say.

Reply:
Thanks, done.